.github/workflows/schedule_candidate_release.yml

```diff
@@ -10,24 +10,26 @@ jobs:
   tag_release:
     name: "Tag candidate release"
     runs-on: ubuntu-20.04
-    # Don't run this in everyone's forks.
-    if: github.repository == 'iree-org/iree'
+    # Don't run the cron in everyone's forks
+    if: github.event_name != 'schedule' || github.repository == 'iree-org/iree'
```

Comment:
Is the intention here to allow the manual trigger on the forked repo?

Reply:
Yes. Doesn't hurt anything and makes testing easier